open-metadata-implementation/admin-services/admin-services-api/src/main/java/org/odpi/openmetadata/adminservices/configuration/registration/AccessServiceDescription.java

```diff
@@ -105,6 +105,12 @@
                                       "digital-architecture",
                                       "Design of the digital services for an organization",
                                       "https://egeria.odpi.org/open-metadata-implementation/access-services/digital-architecture/"),
+    DIGITAL_SERVICE_OMAS            ( 1024,
```

Comment:
1024 is already taken

Reply:
resolved

Reply:
This was a merge error, I've written a test which just checks if these values are unique, but I'll use a separate commit because of possible merge conflicts

Reply:
added the new test and fixed 2 AccessServiceCodes